apps/riscv-tests/isa/rv64uv/Makefrag

```diff
@@ -107,10 +107,10 @@ rv64uv_sc_tests = vadd \
                   vfncvt \
                   vmand \
                   vmnand \
-                  vmandnot \
+                  vmandn \
```

Comment:
@quswar-abid Please retain the vmandnot test as well in the regression/test-suite as it is still retained as alias.